src/ekore/operator_matrix_elements/unpolarized/time_like/as1.py

```diff
@@ -0,0 +1,101 @@
+"""The unpolarized, time-like |NLO| matching conditions."""
+
+import numba as nb
+import numpy as np
+
+from eko.constants import CF
+
+
+@nb.njit(cache=True)
+def A_hg(N, L):
+    r"""Compute the |NLO| heavy-gluon |OME|.
+
+    Implements :eqref:`27` from :cite:`Cacciari:2005ry`.
+
+    Parameters
+    ----------
+    N : complex
+        Mellin moment
+    L : float
+        :math:`\ln(\mu_F^2 / m_h^2)`
+
+    Returns
+    -------
+    A_hg : complex
+        |NLO| heavy-gluon |OME|
+        :math:`A_{hg}^{S,(1)}`
+
+    """
+    result = (
+        2
+        * CF
+        * (
+            (2 + N + N**2) / (N * (N**2 - 1)) * (L - 1)
+            + 4 / (N - 1) ** 2
+            - 4 / N**2
+            + 2 / (N + 1) ** 2
+        )
+    )
+    return result
+
+
+@nb.njit(cache=True)
+def A_gg(L):
+    r"""Compute the |NLO| gluon-gluon |OME|.
+
+    Implements the Mellin transform of
+    :eqref:`24` from :cite:`Cacciari:2005ry`.
+    It is identical to the one in
```

Comment:
why don't we import from there directly then?

Reply:
This is to be consistent with the approach we also followed in ADs whereby at LO where, there were two functions which were identical but we didn't import them. Alessandro and Felix were okay with these since these were small duplications and my rational for this was as follows: 1. it keeps the time_like part self contained and 2. it allows citing the proper time-like papers which also contain these expressions.